tests/e2e/mapper/mapper_suite_test.go

```diff
@@ -0,0 +1,251 @@
+package mapper
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/kubeedge/kubeedge/tests/e2e/utils"
+	"github.com/kubeedge/kubeedge/tests/performance/common"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"io/ioutil"
+	v1 "k8s.io/api/apps/v1"
+	"net/http"
+	"os/exec"
+	"path"
+	"testing"
+	"time"
+)
```

Comment:
please order the imports you can use an empty line to separate them.

Reply:
+1

Reply:
Updated